BackofficeBundle/Form/Type/AreaType.php

```diff
@@ -60,15 +64,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'label' => 'open_orchestra_backoffice.form.area.bo_percent'
         ));
         if(!array_key_exists('disabled', $options) || $options['disabled'] === false){
-            $builder->addEventSubscriber(new AreaCollectionSubscriber($this->areaClass));
+            $builder->addEventSubscriber(new AreaCollectionSubscriber($this->areaClass, $this->translator));
```

Comment:
It's not the only case where we use a new instead of a service. I've made a ticket for that as it's a general problem (https://trello.com/c/VEyutJDJ/1101-plusieurs-objets-sont-crees-avec-des-news-plutot-qu-en-passant-par-un-service-generaliser-l-appel-aux-services-pourra-necessiter)
As this PR is already consequent and there's some other cases of this, I suggest to put these refacto on a separated PR.